ocaml/xapi/cli_frontend.ml

```diff
@@ -1995,7 +1995,7 @@ let rec cmdtable_data : (string*cmd_spec) list =
     };
     "vdi-list-changed-blocks",
     {
-      reqd=["vdi-from"; "vdi-to"];
```

Comment:
If this has never been released, I am happy to have it merged, otherwise is a CLI breaking change

Reply:
@mseri this has never been released.